stable/postgresql/templates/statefulset-slaves.yaml

```
@@ -70,11 +70,9 @@ spec:
           - sh
           - -c
           - |
+            mkdir -p {{ .Values.persistence.mountPath }}/data && chmod 0700 {{ .Values.persistence.mountPath }}/data
```

Can you do it using a single command??
```suggestion
            mkdir -p {{ .Values.persistence.mountPath }}/data && chmod 0700 {{ .Values.persistence.mountPath }}/data
            mkdir -m 700 -p {{ .Values.persistence.mountPath }}/data
```

---

done